libs/services/designer-client-services/src/lib/standard/run.ts

```diff
@@ -263,7 +263,7 @@ export class StandardRunService implements IRunService {
         : response;
 
     return Object.keys(dictionaryResponse).reduce((prev, current) => {
-      return { ...prev, [current]: { displayName: current, value: dictionaryResponse[current] } };
+      return { ...prev, [current]: { displayName: current, value: dictionaryResponse[current].content ?? dictionaryResponse[current] } };
```

Comment:
Is this null ref safe? What if dictionaryresponse[current] is undefined

Reply:
`dictionaryresponse[current]` can't be undefined because `dictionaryResponse` its the object that gets iterated and `current` its the current key in the iteration. What can be undefined it's the `content` property but if that's the case will only return `dictionaryResponse[current]`